frontend/src/components/pages/events/EventDetails.tsx

```diff
@@ -359,13 +359,15 @@ const EventDetails: React.FC<Props> = ({ eventId }) => {
                     </Typography>
                   </>
                 )}
-                <Button
-                  variant="text"
-                  href={calendarFile(event.title, event.startTime, event.endTime, event.location, event.description)}
-                  download="event.ics"
-                >
-                  Last ned i kalender
-                </Button>
+                <Box>
```

Comment:
Should arguably refactor this and place all items here in a Grid instead, however, might be beyond the scope of this PR